eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/admin/handler/GrpcClientHandler.java

```diff
@@ -103,7 +103,7 @@ void delete(HttpExchange httpExchange) throws IOException {
             Error error = new Error(e.toString(), stackTrace);
             String result = JsonUtils.toJSONString(error);
             httpExchange.sendResponseHeaders(500, result.getBytes().length);
-            out.write(result.getBytes());
+            log.error(result);
```

Comment:
After the replacement, does result.getBytes().length not need to be calculated?

Reply:
Seems you're right. Since there's no response body, the length should be 0.

Reply:
BTW, the CI took too much time to run...

Reply:
> BTW, the CI takes too much time to run...

Thanks for feedback.

Reply:
resolved.